app/Widgets/Literacy/Translations/en/widget.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * This file is part of Digital Atlas.
+ *
+ * Digital Atlas is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Digital Atlas is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Levi Costello <leviwcostello@gmail.com>
+ * @license http://opensource.org/licenses/gpl-license.php GNU Public License
+ *
+ */
+/**
+ * Localization strings for the Literacy widget.
+ */
+return [
+    'loading'   =>  'Loading...',
+    'title'     =>  '2020 Literacy',
```

Comment:
Can you change the title to **Literacy Rate**.  You can remove the 2020 as well.

Also, for total can you change it to **Literacy Rate**.  The word Total is not necessary.